docs/database/database-spans.md

```diff
@@ -478,6 +480,36 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Context propagation
+
+**Status**: [Development][DocumentStatus]
+
+### SQL commenter
+
+Instrumentations MAY propagate context using [SQL commenter](https://google.github.io/sqlcommenter/spec/) by injecting comments into SQL queries before execution. Context propagation SHOULD NOT be enabled by default, but instrumentation MAY allow users to opt into it.
+
+The instrumentation implementation SHOULD **append** the comment to the end of the query. Semantic conventions for individual database systems MAY specify different format depending on the specific database system's requirements or preferences. For instance, the instrumentation MAY **prepend** the comment at the beginning of the query.
+
+The instrumentation SHOULD allow users to pass a propagator to overwrite the global propagator.
+
+If the propagator does not provide any value, the instrumentation SHOULD NOT inject any comment into the query.
+
+**Examples:**
+
+- For a query `SELECT * FROM songs` with W3C TraceContext propagator:
+
+  ```sql
+  SELECT * FROM songs /*traceparent='00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01',tracestate='congo%3Dt61rcWkgMzE%2Crojo%3D00f067aa0ba902b7'*/
+  ```
+
+  Note that adding high cardinality comments, like `traceparent` and `tracestate`, to queries can impact the performance for some database systems, like MySQL. See [this related issue](https://github.com/google/sqlcommenter/issues/284) for more details.
```

Comment:
Severely affected databases are Oracle and SQL Server for both prepared and non-prepared statements.